o4/o4_prune.py

```diff
@@ -0,0 +1,52 @@
+
+from pathlib import Path
+from time import time
+
+from o4_fstat import prune_fstat_cache
+
+
+def prune_data(o4_dir):
+    prune_fstat_cache(o4_dir)
+    prune_cleaned(o4_dir)
+
+
+def prune_cleaned(o4_dir):
+    '''
+    Remove all read-only files from an abandoned cleaned directory
+    (unless it's too recent).
+    '''
+    cleaned = Path(o4_dir) / 'cleaned'
```

Comment:
It overrides `__div__` ?

Reply:
Yes, that's the equivalent of os.path.join